.github/workflows/continuous-integration.yml

```
@@ -118,3 +118,32 @@ jobs:
         run: yarn --immutable
       - name: Validate dynamic side effects
         run: yarn build:dynamic-side-effects -- --check
+
+  Check_PackageIndexFiles:
+    name: Check Package Index Files
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@v4
+      - name: Use Node.js v20
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: yarn
+          registry-url: https://registry.yarnpkg.org/
+      - name: Install Dependencies
+        run: yarn --immutable
+      # It would be nice to cache turbo's output, but it passes for all subsequent runs since it doesn't commit to fs
+      - name: Generate index files
```

```suggestion
      - name: Generate index files
```

I don't really like these kind of code comments. It's not useful to add.